docs/self-managed/platform-deployment/helm-kubernetes/platforms/amazon-eks/dual-region.md

```diff
@@ -0,0 +1,534 @@
+This allows for easier communication between the two clusters by forwarding DNS queries from the region 0 cluster to the region 1 cluster and vice versa.
+
+<CoreDNSKubeDNS />
+
+You are configuring the CoreDNS from the cluster in **Region 0** to resolve certain namespaces via **Region 1** instead of using the in-cluster DNS server. Camunda applications (e.g. Zeebe brokers) to resolve DNS record names of Camunda applications running in another cluster.
+
+#### Internal load-balancer
+
+1. Expose `kube-dns`, the in-cluster DNS resolver via an internal load-balancer in each cluster.
+
+```bash
+kubectl --context $CLUSTER_0 apply -f https://github.com/camunda/c8-multi-region/blob/aws-operational/test/resources/aws/2-region/kubernetes/internal-dns-lb.yml
+kubectl --context $CLUSTER_1 apply -f https://github.com/camunda/c8-multi-region/blob/aws-operational/test/resources/aws/2-region/kubernetes/internal-dns-lb.yml
+```
+
+2. Retrieve the hostnames of the load-balancer and get their internal IP addresses.
+
+```bash
+# execute on cluster-london
+HOST1=$(kubectl --context $CLUSTER_0 -n kube-system get svc across-cluster-dns-tcp -o jsonpath="{.status.loadBalancer.ingress[0].hostname}" | cut -d - -f 1)
+
+# execute on cluster-paris
+HOST2=$(kubectl --context $CLUSTER_1 -n kube-system get svc across-cluster-dns-tcp -o jsonpath="{.status.loadBalancer.ingress[0].hostname}" | cut -d - -f 1)
+```
+
+The following will print the IP addresses of the internal load-balancer in each cluster, which you will use to configure the opposite cluster for DNS resolving:
+
+```bash
+# gives you the IPs from region London that need to be configured in Paris
+aws ec2 describe-network-interfaces --region $REGION_0 --filters Name=description,Values="ELB net/${HOST1}*" --query  'NetworkInterfaces[*].PrivateIpAddress' --output text
+# Output example
+# 10.190.114.39   10.190.172.21   10.190.134.196
+
+# gives you the IPs from region Paris that need to be configured in London
+aws ec2 describe-network-interfaces --region $REGION_1 --filters Name=description,Values="ELB net/${HOST2}*" --query  'NetworkInterfaces[*].PrivateIpAddress' --output text
+# Output example
+# 10.202.30.193   10.202.46.72   10.202.65.75
+```
+
+#### CoreDNS configuration
+
+In the next part, you will configure CoreDNS to forward DNS queries for specific namespaces to the other cluster.
+
+You have to choose unique namespaces for Camunda 8 installations. The namespace for Camunda 8 installation in the cluster of region 0, needs to have a different name from the namespace for Camunda 8 installation in the cluster of region 1. This is required for proper traffic routing between the clusters.
+
+For example, you can install Camunda 8 into the `camunda-london` namespace in the `cluster-london` cluster, and `camunda-paris` namespace on the `cluster-paris` cluster.
+Using the same namespace on both clusters won't work as CoreDNS won't be able to distinguish between traffic targeted at the local and remote cluster.
```

Comment:
```suggestion
Using the same namespace names on both clusters won't work as CoreDNS won't be able to distinguish between traffic targeted at the local and remote cluster.
```
I know it's a bit of tautology, but just to ensure people understand it's not about not installing Camunda twice into a same namespace, but also about not having 2 samely named namespaces in different clusters.